trp_project_scrum/security/ir_rules_collaborators_followers.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="0">
+        <record id="project_rule_foll" model="ir.rule">
+            <field name="name">Show only collaborator follower Project</field>
+            <field name="model_id" ref="project.model_project_project"/>
+            <field name="domain_force">[
+                ('message_follower_ids', 'in', [user.partner_id.id]),
+                ('use_scrum', '=', True)
+                ]</field>
+            <field name="groups" eval="[(4,ref('trp_project_scrum.collaborators_followers'))]"/>
+            <field name="global" eval="False" />
+            <field name="perm_unlink" eval="False" />
+            <field name="perm_write" eval="False" />
+            <field name="perm_read" eval="True" />
+            <field name="perm_create" eval="False" />
+        </record>
+
+        <record id="project_task_rule_foll" model="ir.rule">
+            <field name="name">Show Only collaborator Follower Task</field>
+            <field name="model_id" ref="project.model_project_task"/>
+            <field name="domain_force">[
+                    ('project_id.message_follower_ids', 'in', [user.partner_id.id]),
+                    ('use_scrum', '=', True)]</field>
+            <field name="groups" eval="[(4,ref('trp_project_scrum.collaborators_followers'))]"/>
+            <field name="global" eval="False" />
+            <field name="perm_unlink" eval="False" />
+            <field name="perm_write" eval="True" />
+            <field name="perm_read" eval="True" />
+            <field name="perm_create" eval="False" />
+        </record>
+
+        <record id="project_task_type_rule_foll" model="ir.rule">
```

Comment:
but i consider task types harmless, so this doesn't really need a restriction imho